src/baseline_model.py

```diff
@@ -0,0 +1,33 @@
+from src import yodict
+from src.eyo import Eyo
+from src.model import yoModel
+from tqdm import tqdm
+from typing import Tuple, List
+
+
+class Baseline(yoModel):
+
+    def __init__(self) -> None:
+        super().__init__()
+        
+        yo_dict = yodict.get_safe()
+        self.model = Eyo(yo_dict)
+    
+    def _predict(self, text: str) -> List[Tuple[int, int]]:
+        """Fuctuinon for yoword substring predictions by single string"""
```

Comment:
```suggestion
        """Function for yoword substring predictions by single string"""
```

Хорошей практикой именования функций считается постановка в форму глагола, отвечающую на вопрос: "Что сделать?" или "Что делает?". К примеру, [вот чуть подробнее](https://google.github.io/styleguide/pyguide.html#383-functions-and-methods) про оформление докстрингов у Google.

Тогда докстринг мог бы выглядеть примерно так:
```suggestion
        """Predicts yo word substrings for a single string"""
```